1e/README.md

```diff
@@ -0,0 +1,52 @@
+# 1E
+
+## Overview
+
+The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. These metrics are visualized on the out-of-the-box dashboard, and can also be used to  create your own visualizations. 
```

Comment:
Non-blocking suggestion but it would be good to add one line about what 1E does.

```suggestion
1E is a digital experience solution that automatically surfaces user experience issues with recommended troubleshooting steps, reducing points of friction. The 1E Datadog Integration is an API-based integration that collects metrics from 1E products and forwards them to your Datadog account. These metrics are visualized on the out-of-the-box dashboard, and can also be used to  create your own visualizations. 
```